src/libData/AccountData/AccountStoreSC.tpp

```diff
@@ -315,51 +323,12 @@ bool AccountStoreSC<MAP>::UpdateAccounts(const uint64_t& blockNum,
         if (ret) {
           std::string runnerPrint;
 
-          pid = -1;
-          auto func2 = [this, &runnerPrint, &ret, &pid, gasRemained, &receipt,
-                        &is_library, &extlibs_exports]() mutable -> void {
-            try {
-              if (!SysCommand::ExecuteCmd(
-                      SysCommand::WITH_OUTPUT_PID,
-                      GetCreateContractCmdStr(m_root_w_version, is_library,
-                                              gasRemained, 0),
-                      runnerPrint, pid)) {
-                LOG_GENERAL(WARNING,
-                            "ExecuteCmd failed: " << GetCreateContractCmdStr(
-                                m_root_w_version, is_library, gasRemained, 0));
-                receipt.AddError(EXECUTE_CMD_FAILED);
-                ret = false;
-              }
-            } catch (const std::exception& e) {
-              LOG_GENERAL(WARNING,
-                          "Exception caught in SysCommand::ExecuteCmd (2): "
-                              << e.what());
-              ret = false;
-            }
-
-            cv_callContract.notify_all();
-          };
-          DetachedFunction(1, func2);
-
-          {
-            std::unique_lock<std::mutex> lk(m_MutexCVCallContract);
-            cv_callContract.wait(lk);
-          }
+          // invoke scilla runner
+          InvokeInterpreter(RUNNER_CREATE, runnerPrint, scilla_version,
+                            is_library, gasRemained, amount, ret, receipt);
```

Comment:
It seems like in the original code we called runner with balance 0, whereas now we're using `amount` for the balance. Is this a code fix?

Reply:
yes you can say that, but during deployment the amount in the command doesn't really matter, so we were fine in the past